src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,220 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //-------------------------------------------------
+  //Main Container
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-3xs; // allow space for focus styles
+  }
+
+  //-------------------------------------------------
+  //Main Container > Table Header
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-md;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //-------------------------------------------------
+  //Main Container > Table (Default Styles)
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    thead {
+      @include type-style('body-short-01');
+      background-color: $ui-03;
+      font-weight: 700;
+    }
+
+    tbody {
+      @include type-style('body-short-01');
+      background-color: $ui-01;
+      width: 100%;
+    }
+
+    tr {
+      border: none;
+      height: rem(48px);
+      width: 100%;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) {
+      background: $hover-field;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) td {
+      color: $text-01;
+      background: $hover-field;
+      border-bottom: 1px solid $hover-field;
+      border-top: 1px solid $hover-field;
+    }
+
+    th,
+    td {
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+      vertical-align: top;
+      text-align: left;
+      color: $text-02;
+      border-top: 1px solid $ui-01;
+      border-bottom: 1px solid $ui-01;
+    }
+
+    th {
```

Comment:
Might help to refactor into:

```scss
    th,
    td {
      padding-left: $spacing-sm;
      padding-right: $spacing-sm;
      vertical-align: top;
      text-align: left;
    }

     th {
      color: $text-01;
      border-top: 1px solid $ui-03;
      border-bottom: 1px solid $ui-03;
    }

    td {
      color: $text-02;
      border-top: 1px solid $ui-01;
      border-bottom: 1px solid $ui-01;
    }
```

Just because the color/border properties aren't shared